BoardConfig.mk

```diff
@@ -96,11 +106,6 @@ BOARD_USES_DRM_HWCOMPOSER := true
 
 BOARD_USES_METADATA_PARTITION=true
 
-# Verified boot
-ifeq ($(ENABLE_AVB),true)
-  BOARD_AVB_ENABLE := true
-endif
-
```

Comment:
If this block was moved 40 lines above for some reason, then it's better to comment that it need to be placed before something.
If there is no specific reason to move block - may be it can be left in old place.
Not comment, just suggection.

Reply:
This block will be changed in new patch version